eng/scripts/Logging-Functions.ps1

```diff
@@ -10,6 +10,26 @@ function LogInfo {
   Write-Host "$args"
 }
 
+function LogNotice {
+  if (Test-SupportsGitHubLogging) {
+    Write-Host ("::notice::$args" -replace "`n", "%0D%0A")
+  }
+  else {
+    # No equivalent for DevOps
+    Write-Host "[Notice] $args"
+  }
+}
+
+function LogNoticeForFile($file, $noticeString) {
+  if (Test-SupportsGitHubLogging) {
+    Write-Host ("::notice file=$file,line=1,col=1::$noticeString" -replace "`n", "%0D%0A")
```

Comment:
Is notice just an informational annotation?

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-notice-message

`notice` is between `debug` and `warning` in severity.  It was perfect for this case, where we want to "notify" spec authors about converting to TypeSpec, but we don't want to make it a full `warning` until later.  But it still appears in the git diff UI, it's listed in the summary of the Action results, etc.